Dockerfile.windows

```diff
@@ -67,20 +68,20 @@ RUN cat sha256sum.txt | head -n1 | awk '{print $1"  wins.exe"}' >> wins.exe.sha2
 RUN sha256sum -c wins.exe.sha256
 RUN mv wins.exe rancher/
 
-RUN curl -sLO https://dl.k8s.io/release/${KUBERNETES_VERSION}/bin/windows/amd64/kubectl.exe
-RUN curl -sLO https://dl.k8s.io/${KUBERNETES_VERSION}/bin/windows/amd64/kubectl.exe.sha256
+RUN curl -sLO https://dl.k8s.io/release/$KUBERNETES_VERSION/bin/windows/amd64/kubectl.exe
```

Comment:
What's the benefit of removing the curly braces?

Reply:
trying to confirm env vs arg, dockerfile syntax vs shell syntax... we had issues in drone files i just dont remember which way it works yet. still a wip and i can't get it to build locally

Reply:
> trying to confirm env vs arg, dockerfile syntax vs shell syntax... we had issues in drone files i just dont remember which way it works yet. still a wip and i can't get it to build locally

should be safe to leave the braces alone